src/FontStashSharp/FontGlyph.cs

```diff
@@ -35,7 +35,8 @@ public bool IsEmpty
 	public class DynamicFontGlyph: FontGlyph
 	{
 		public int Size;
-		public IFontSource Font;
+		public IDynamicFontSource Font;
+		public uint KernelWidth;

```

Comment:
Please, change it to ordinary int in order to keep the coding style consistent.
Btw, do we really need this property?
As it seems that it is being set to 2 and never changed thereafter.
Maybe introduce a constant instead?

Reply:
I think you're right here, for the kernel width it doesn't make too much sense to have it flexible. Will change that stuff to a constant